web/src/lib/utils.ts

```diff
@@ -25,6 +25,24 @@ export const WorkingHours = [
   '17:00',
 ] as const;
 
+export enum Links {
+  STUDENT,
+  TEACHER,
+  CLASS,
+  ROOM,
+  STUDENT_GROUP,
+  GRADE,
+  CURRICULUM,
+}
```

Comment:
Maybe I am blind but where's this enum used?

Reply:
You're not blind, it is no longer needed and I forgot to remove it. Thank you